crates/bevy_math/src/sampling/shape_sampling.rs

```diff
@@ -624,7 +627,7 @@ mod tests {
         for _ in 0..5000 {
             let point = circle.sample_boundary(&mut rng);
 
-            let angle = f32::atan(point.y / point.x) + PI / 2.0;
+            let angle = ops::atan(point.y / point.x) + PI / 2.0;
```

Comment:
Seems like a prime candidate for `atan2`.

Reply:
I agree, but it's currently written with the angle lying in the range [0, PI] in mind, so changing it to `atan2` basically requires rewriting it, which I'd rather not do right now.